run_dir/static/js/data_del_plot.js

```diff
@@ -0,0 +1,357 @@
+window.current_plot_data = null;
+window.current_plot_obj = null;
+window.current_view_schemes = null;
+
+$(function(){
+    init_page_js();
+});
+
+function refresh_plot(){
+    var params = get_parameters();
+    if (window.current_plot_data == null){
+        get_plot_data(key=params[0], name=params[1], view_type=params[2], search_string=params[3], inst_type_filter=params[4]);
+    }else{
+        make_plot(key=params[0], name=params[1], view_type=params[2], inst_type_filter=params[4]);
+    }
+}
+
+function make_plot(key, name, view_type, filter_inst_type){
+    var toplot = {
+        chart: {
+            type: 'scatter',
+            events: {
+                render: function(){
+                    let series = this.series
+                    let sum = 0
+                    for (let i = 0; i < series.length; i++){
+                        if (series[i].visible){
+                            for (let j = 0; j < series[i].data.length; j++){
+                                sum += series[i].data[j].y
+                            }
+                        }
+                    }
+                    if (sum === 0) return '0 Bytes';
+                    const k = 1024;
+                    const dm = 2 < 0 ? 0 : 2;
+                    const sizes = ['Bytes', 'KB', 'MB', 'GB', 'TB', 'PB', 'EB', 'ZB', 'YB'];
```

Comment:
This code is used twice, so nicer to move it out into its own function to purely take an integer and return the nicely formatted value. Then can just call that.

Reply:
Fixed it!